packages/edit-site/src/components/template-switcher/style.scss

```diff
@@ -46,3 +47,14 @@
 		display: block;
 	}
 }
+
+.edit-site-template-switcher__theme-preview-name {
+	font-weight: 500;
+	font-size: 140%;
+}
+
+.edit-site-template-switcher__theme-preview-screenshot {
+	margin-bottom: 1em;
```

Comment:
I haven't really seen us use `em` units elsewhere. I think we might wanna look at using the grid unit variables: https://github.com/WordPress/gutenberg/blob/5494ff5e528fa7ebe2cb53bf33cf914de3405e60/packages/base-styles/_variables.scss#L27

Reply:
Ah, my bad, not really familiar with Gutenberg's styling. I'll change to use use `$grid-unit`.

Reply:
`$grid-unit-15` looked good to me; made the change in d084f58.